src/applications/mhv-landing-page/tests/e2e/va-gov-health-tools/va-gov-health-tools.cypress.spec.js

```diff
@@ -32,7 +32,8 @@ describe(`${appName} -- VA.gov Health Tools feature`, () => {
         };
         cy.findByRole('heading', heading).should.exist;
         links.forEach(({ text, href }) => {
-          cy.findByRole('link', { name: text }).should(
+          const linkNameRegex = new RegExp(text, 'i');
```

Comment:
Note: This results in `/text/i`

Reply:
Yes, but should the match be case insensitive?

Reply:
I can remove it.